helm/prometheus-rules/templates/grafana-agent-rules-configmap.yaml

```diff
@@ -1,41 +1,13 @@
 {{- if .Values.mimir.enabled }}
 apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: alloy-rules-config
+  namespace: giantswarm
 data:
   values: |
-    networkPolicy:
-      cilium:
```

Comment:
Why did you remove this policy configuration ?

Reply:
Because the networkpolicy is directly within the helm chart of Alloy.

Reply:
We need to push it  upstream then because @marieroque will not like this :D

Reply:
Maybe we need to add the CNP:
https://github.com/giantswarm/grafana-agent-app/blob/main/helm/grafana-agent/templates/cilium-networkpolicy.yaml
as manifest in that repo ?

Reply:
@TheoBrigitte did your tests work in term of communication ?
Did Hubble UI show no dropped request ?

Reply:
I did not check Hubble UI, my test was successful (I deleted all the rules from Mimir, and Alloy re-imported them all)
Also there is a CNP already added in Alloy chart > https://github.com/giantswarm/alloy-app/blob/main/helm/alloy/templates/cilium-networkpolicy.yaml

Reply:
Just checked and there's no error in Hubble UI :heavy_check_mark: